config/routes.rb

```diff
@@ -49,6 +49,7 @@
     get 'reservation/:id/reject', to: 'reservations#reject', as: :reservation_reject
 
     get 'nook/check_availability/:reservation_id', to: 'nooks#check_availability', as: :check_availability
+    delete 'nooks/:id/remove_photo/:photo_id', to: 'nooks#remove_photo', as: :remove_photo
```

Comment:
Can we make this route more RESTful? Something like `delete 'nooks/:id/photos/:photo_id'`